services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java

```diff
@@ -45,40 +49,69 @@
     @Autowired
     private RouteDatabaseService routeDatabaseService;
 
+    @Autowired
+    private RouteWithVpcMapperService routeWithVpcMapperService;
+
+    @Autowired
+    private RouteWithSubnetMapperService routeWithSubnetMapperService;
+
     @RequestMapping(
             method = GET,
-            value = {"/vpcs/{vpcId}/routes/{routeId}"})
-    public RouteWebJson getRuleByVpcId(@PathVariable String vpcId, @PathVariable String routeId) throws Exception {
+            value = {"routes/vpcs/{vpcId}/get"})
```

Comment:
@kevin-zhonghao We may need to change the API doc if any.

Reply:
It seems we didn't have route API doc right now, I will add it after